### PR TITLE
New extension: dev.to

### PR DESCRIPTION
**Pull request by aeither**
## Description

Search Reading List. It is the list of all the articles saved in dev.to.
Search Articles. Explore and manage all the articles published by you or still in draft. Update the articles.
Create Article. Start a new article directly from Raycast.

## Screencast

1-reading-list

2-articles

3-create-article

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/google-maps-search/package.json [hunk 1]
```diff
@@ -5,6 +5,9 @@
   "description": "Quickly open Google Maps directions, for example from your current location to your home.",
```
Comment (sxn) [inline extensions/google-maps-search/package.json:5]: I'm not sure why `google-maps-search` extension was modified 😅 could you update the branch to make sure your PR only touches the `dev-to` extension?

File: extensions/dev-to/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+## Changelog
+
+The [changelog](CHANGELOG.md) documents all notable updates to the extension.
+
+## Contributing
+
+See [Contribute to an Extension](https://developers.raycast.com/basics/contribute-to-an-extension) if you're interested in contributing to this extension!
+
+Feel free to open an issue [open an issue with feedback](https://github.com/aeither/extensions/issues)!
```
Comment (sxn) [inline extensions/dev-to/README.md:40]: the `command-icon.png` seems like a leftover from the template project 🤔 mind removing it?

**Comment by sxn** [comment]
hey @aeither - thanks for the PR! I left a couple of comments w/ some cleanup that needs to be done. Additionally, I've encountered an issue: I can't open draft articles; as I always get a 404.

https://user-images.githubusercontent.com/1155589/186439879-2ac5fb91-d837-4bb8-b6f6-f7f1f402b4cb.mov

Ping me once you've had a chance to fix these issues and I'll re-review the PR 😺

**Comment by aeither** [comment]
Hi @sxn , I have fixed the issue with draft articles and cleaned up a bit. It is ready for another review. Thanks.

**Comment by aeither** [comment]
@sxn Just want to mention it is still labeled. status: awaiting response from dev

**Comment by sxn** [comment]
🎉

**Pull request merged**